.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+name: 'ci'
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  ci:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: 'actions/checkout@v2'
+
+      - name: 'Use Node.js'
+        uses: 'actions/setup-node@v2.4.0'
+        with:
+          node-version: 'lts/*'
+
+      - name: 'Cache Node dependencies'
+        uses: 'actions/cache@v2.1.6'
+        with:
+          path: '~/.npm'
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
```

```suggestion
      - name: 'Use Node.js'
        uses: 'actions/setup-node@v2.4.0'
        with:
          node-version: 'lts/*'
          cache: 'npm'
```

Nowadays its enough[ to do `cache: 'npm'`](https://github.com/actions/setup-node/releases/tag/v2.2.0) 🎉

---

That's not working without `package-lock.json` badly.

---

Oh, had totally missed that part, never mind me then